Automattic/newspack-ads
# refactor: use namespaces

This PR is part of #332. It implements the use of namespaces for better code readability and maintenance.

No changes should take place. To make sure all features continue to work as expected:

Check out https://github.com/Automattic/newspack-plugin/pull/1571

1. Tests should pass
2. Install SCAIP, Broadstreet and connect with GAM (through proxied OAuth or Service Account)
3. Enable header bidding through `define( 'NEWSPACK_ADS_EXPERIMENTAL_BIDDERS', true );`
4. Add a GAM and a Broadstreet unit to placements
5. Add mock IDs to header bidding with debug mode on
6. Add an ad unit block to a page
7. Add an ad unit widget to a sidebar
8. Modify settings through the wizard and make sure they persist after page refresh
9. Navigate through pages with ads and make sure the ads are displayed as expected the header bidding logs show bid attempts
